scripts/release_ppa.sh

```diff
@@ -156,7 +156,7 @@ cp "/tmp/${packagename}_${debversion}.orig.tar.gz" ../
 # Create debian package information
 
 mkdir debian
-echo 9 > debian/compat
+echo 13 > debian/compat
```

Comment:
We have one more script for building PPA packages. I think it should be updated too:
https://github.com/ethereum/solidity/blob/83002f3a0e6276c3e4056d2413bd3008e0a336cb/scripts/deps-ppa/static_z3.sh#L76

Reply:
done